irr/include/irrString.h

```diff
@@ -166,21 +167,29 @@ class string
 
 	//! Assignment operator for strings, ASCII and Unicode
 	template <class B>
-	string<T> &operator=(const B *const c)
+	string<T> &operator=(const B * c)
```

Comment:
```suggestion
	string<T> &operator=(const B *c)
```

(Also: Huh why remove this `const`? I don't mind it, but I don't see why we need to either.)

Reply:
I will fix this my son.

Reply:
I appreciate it father